packages/react-charting/jest.config.js

```diff
@@ -1,17 +1,28 @@
 let { createConfig, resolveMergeStylesSerializer } = require('@fluentui/scripts/jest/jest-resources');
 let path = require('path');
 
+function getEsmOnlyPackagesToCjsMapping() {
+  /**
+   * relative path to monorepo root
+   */
+  const prefix = `<rootDir>/../../`;
+  /**
+   * map of packages that ship only as ESM
+   */
+  const cjsPathsToEsmOnlyPackages = {
+    '^d3-scale$': prefix + 'node_modules/d3-scale/dist/d3-scale.js',
```

Comment:
Are the other modules in the list not required?

Reply:
other approach is to transpile those ESM libraries on our side. I tried that but was getting invalid snapshots. We can chat further about this appraoch.